src/theory/arith/nonlinear_extension.cpp

```diff
@@ -1407,6 +1386,53 @@ int NonlinearExtension::checkLastCall(const std::vector<Node>& assertions,
     return lemmas_proc;
   }
 
+  // process SINE phase shifting
+  for (const Node& a : trig_no_base)
+  {
+    if (d_trig_base.find(a) == d_trig_base.end())
+    {
+      Node y = NodeManager::currentNM()->mkSkolem(
+          "y",
+          NodeManager::currentNM()->realType(),
+          "phase shifted trigonometric arg");
+      Node new_a = NodeManager::currentNM()->mkNode(a.getKind(), y);
```

Comment:
This is moved code, but since there is not that much moved code in the PR you could just revisit this little part, cache ``NodeManager::currentNM()``, and make sure this code is ok.